src/Microsoft.AspNetCore.ApplicationInsights.HostingStartup/ApplicationInsightsStartupLoader.cs

```diff
@@ -22,6 +24,16 @@ public class ApplicationInsightsHostingStartup : IHostingStartup
         public void Configure(IWebHostBuilder builder)
         {
             builder.UseApplicationInsights();
+            builder.ConfigureServices(InitializeServices);
+        }
+
+        /// <summary>
+        /// Adds the Javascript <see cref="TagHelperComponent"/> to the <see cref="IServiceCollection"/>.
+        /// </summary>
+        /// <param name="services">The <see cref="IServiceCollection"/> associated with the application.</param>
+        protected virtual void InitializeServices(IServiceCollection services)
```

Comment:
Yeah it's private now. I'll update the PR in a bit. Also I have removed the new sample I added and added a `View` and a `Controller` to the existing sample.